### PR TITLE
Improve userlist dropdown menu key-board functionality

### LINKED ISSUE #5394: Dropdown does not close when you select another user in userlist
bug-userlist

### PR DESCRIPTION
**Pull request by KDSBrowne**
This fixes #5394 

* up / down arrow -> cycle through users in userlist (if menu is closed)
* up / down arrow  -> cycle through open menu options (if menu is open)
* Removes box shadow on the userlist item with an open menu

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-list-item/user-list-content/component.jsx [hunk 1]
```diff
@@ -120,6 +122,10 @@ class UserListContent extends Component {
 
     const scrollContainer = this.props.getScrollContainerRef();
     scrollContainer.removeEventListener('scroll', this.handleScroll, false);
+
+    setTimeout(() => {
```
Comment (oswaldoacauan) [inline bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-list-item/user-list-content/component.jsx]: hmmm

File: bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/component.jsx [hunk 2]
```diff
@@ -122,6 +126,14 @@ class UserParticipants extends Component {
     }
   }
 
+  setMenuState(state) {
```
Comment (oswaldoacauan) [inline bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/component.jsx]: `setMenuState` seems to vague. `setDropdownOpenState` seems more reasonable.

File: bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-list-item/user-list-content/component.jsx (hunk 1 above)
Comment (oswaldoacauan) [reply bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-list-item/user-list-content/component.jsx]: did you dig why the race condition happens?

File: bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-list-item/user-list-content/component.jsx [hunk 3]
```diff
@@ -262,7 +263,13 @@ class UserListContent extends Component {
         isOpen={this.state.isActionsOpen}
         onShow={this.onActionsShow}
         onHide={this.onActionsHide}
-        className={cx(styles.dropdown, styles.userListItem, userItemContentsStyle)}
+        className={
+          cx(
+            styles.dropdown,
+            userItemContentsStyle,
+            this.state.isActionsOpen ? styles.usertListItemWithMenu : styles.userListItem,
```
Comment (oswaldoacauan) [inline bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-list-item/user-list-content/component.jsx]: styles.userListItem: this.state.isActionsOpen,

File: bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-list-item/user-list-content/component.jsx [hunk 4]
```diff
@@ -263,13 +266,7 @@ class UserListContent extends Component {
...
         onShow={this.onActionsShow}
         onHide={this.onActionsHide}
-        className={
-          cx(
-            styles.dropdown,
-            userItemContentsStyle,
-            this.state.isActionsOpen ? styles.usertListItemWithMenu : styles.userListItem,
-          )
-        }
+        className={cx(userItemContentsStyle)}
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-list-item/user-list-content/component.jsx]: no need of `cx` for just one class

**Pull request merged**